dotcom-rendering/src/amp/components/elements/TwitterBlockComponent.tsx

```diff
@@ -61,14 +61,11 @@ const TextStyle = (pillar: ArticleTheme) => css`
 
 const makeFallback = (html: string): string | null => {
 	const { window } = new JSDOM(html);
-	const blockquotes = window.document.getElementsByTagName('blockquote');
+	const blockquote = window.document.getElementsByTagName('blockquote')[0];
```

Comment:
Would this be more common?

```suggestion
	const blockquote = window.document.querySelector('blockquote');
```